recipes/seqsero2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "SeqSero2" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "1e677166dba88224ee60f6c4d1585d7f4af4b03b72c89d800bf6fcaf595f3b29" %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/LSTUGA/seqsero2_test/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
You'll want this to be `noarch: generic` instead, so conda will move things around appropriately for anything python version.

Reply:
Okay, thanks!